src/Config/ObjectModel/EntityRelationshipKey.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+
+namespace Azure.DataApiBuilder.Config.ObjectModel
+{
+    /// <summary>
+    /// Identifies a specific value pair:
+    /// 1. entity name
+    /// 2. relationship name
+    /// Which can be used to uniquely identify a relationship (ForeignKeyDefinition object(s)).
+    /// </summary>
+    [DebuggerDisplay("{EntityName} - {RelationshipName}")]
+    public class EntityRelationshipKey
+    {
+        public string EntityName { get; set; }
```

Comment:
From the usages, looks like EntityName is the source entity name for the relationship. Helpful to add a comment here or in the summary above.

Reply:
added comment